### ISSUE TITLE
Maven filtering corrupts sample application .war during build

### ISSUE THREAD

**Issue by jamesfalkner** [Azure/rhel-jboss-templates#53]
It appears that the `eap-session-replication.war` file that is checked into the source repo is being "filtered" (and therefore corrupted) by maven. If I do `mvn clean package` in any of the offer subdirectories it appears to work, but the `.war` file written out into the `target/` directory gets corrupted:

```
➜  eap74-rhel8-byos-multivm git:(master) ✗ unzip -l src/main/scripts/eap-session-replication.war
Archive:  src/main/scripts/eap-session-replication.war
  Length      Date    Time    Name
---------  ---------- -----   ----
        0  05-06-2021 09:12   META-INF/
      122  04-02-2019 11:46   META-INF/MANIFEST.MF
        0  05-06-2021 09:12   META-INF/maven/
        0  05-06-2021 09:12   META-INF/maven/eap-session-replication/
        0  05-06-2021 09:12   META-INF/maven/eap-session-replication/eap-session-replication/
      129  04-02-2019 11:46   META-INF/maven/eap-session-replication/eap-session-replication/pom.properties
      745  04-02-2019 11:14   META-INF/maven/eap-session-replication/eap-session-replication/pom.xml
     6856  05-06-2021 09:22   testHA.jsp
        0  05-06-2021 09:12   WEB-INF/
        0  07-30-2020 17:16   WEB-INF/classes/
      767  04-02-2019 11:24   WEB-INF/web.xml
---------                     -------
     8619                     11 files
➜  eap74-rhel8-byos-multivm git:(master) ✗ unzip -l target/arm/scripts/eap-session-replication.war
Archive:  target/arm/scripts/eap-session-replication.war

caution:  zipfile comment truncated
error [target/arm/scripts/eap-session-replication.war]:  missing 3299648825 bytes in zipfile
  (attempting to process anyway)
error [target/arm/scripts/eap-session-replication.war]:  attempt to seek before beginning of zipfile
  (please check that you have transferred or created the zipfile in the
  appropriate BINARY mode and that you have compiled UnZip properly)
```

**Comment by jamesfalkner** [comment]
There is a stanza in the `azure-javaee-iaas-parent` project's `pom.xml`:

```xml
            <configuration>
              <outputDirectory>${project.build.directory}/arm/scripts</outputDirectory>
              <resources>
                <resource>
                  <directory>src/main/scripts</directory>
                  <filtering>true</filtering>
                </resource>
              </resources>
            </configuration>
```

Which I assume is the culprit. Perhaps the .war file needs to be moved elsewhere?

**Comment by edburns** [comment]
Long term we should make it so the war is built from source.

**Comment by edburns** [comment]
We use the filtering to make it so you can do variable expansion in ARM templates.  It's very handy.

**Comment by edburns** [comment]
https://maven.apache.org/plugins/maven-resources-plugin/examples/filter.html

> If you have both text files and binary files as resources it is recommended to have two separated folders. One folder src/main/resources (default) for the resources which are not filtered and another folder src/main/resources-filtered for the resources which are filtered.

> Which I assume is the culprit. Perhaps the .war file needs to be moved elsewhere?

This is likely the easiest option.

**Comment by edburns** [comment]
Fixed in #54 .
